releasenotes/notes/add-clifford-t-transpiler-pipeline-e640b5ba64eab13a.yaml

```diff
@@ -32,10 +32,28 @@ features_transpiler:
     translation into Clifford+T gates.
 
     It is important to note that the specified Clifford+T basis gate set should be
-    universal, or else transpilation might not suceed. While the gate set 
+    universal, or else transpilation might not succeed. While the gate set 
     ``["h", "t", "tdg"]``or even ``["h", "t"]`` is sufficient for universality,
     it is recommended to add more Clifford gates to the set if possible, as
     otherwise the translation might be less efficient. For example, not including
     S-gate might trigger decomposing S-gates into pairs of T-gates (that is,
     decomposing Clifford gates into non-Clifford gates, which might not be the
     desired behavior).
+
+    Here is an additional slightly larger example::
```

Comment:
2 examples are a bit overkill IMO, I think the top one is good enough (you could add a second gate there if you wanted or something)

Reply:
Shelly was specifically asking for an example over multiple qubits 😄. But actually the two examples are slightly different: the first uses `get_clifford_get_names`, and the second a custom Clifford+T basis.